applications/feedback_sentiment_analyzer/SPECIFICATION.md

```diff
@@ -0,0 +1,224 @@
+# FSA Tech spec
+
+This document contains the technical specifications for Feedback Sentiment Analyzer (FSA), a sample application that showcases AWS services and SDKs.
+
+It explains:
```

Comment:
Architecture should be included as well with a diagram.

Reply:
Add before Inputs and Outputs